website/content/en/docs/upgrading-sdk-version/v1.0.0.md

```diff
@@ -0,0 +1,68 @@
+---
+title: v1.0.0
+weight: 999000000
+---
+
+## Upgrade your project from version "2" to "3-alpha"
+
+The SDK's default Go plugin no longer supports OLM- or scorecard-related project files nor writes a `plugins` PROJECT field for projects scaffolded previously with `operator-sdk init --project-version=2`, Please migrate to project version "3-alpha" for support of these features by adding the following to your `PROJECT` file:
+```yaml version: "3-alpha" # Updated from "2" projectName: <output of $(basename $(pwd))> layout: go.kubebuilder.io/v2 plugins:
+  go.sdk.operatorframework.io/v2-alpha: {}
+```
+
+_See [#3697](https://github.com/operator-framework/operator-sdk/pull/3697) for more details._
+
+## Add the samples scaffold marker to your `config/samples/kustomization.yaml`
+
+Add the `+kubebuilder:scaffold:manifestskustomizesamples` to your `config/samples/kustomization.yaml` file like so (using an example sample file):
+```yaml resources: - cache_v1alpha1_memcached.yaml # +kubebuilder:scaffold:manifestskustomizesamples ```
+
+_See [#3645](https://github.com/operator-framework/operator-sdk/pull/3645) for more details._
+
+## Update your Makefile's `bundle` recipe to inject an operator image tag.
+
+Make the following update to your Makefile's `bundle` recipe, which will allow you to set `make bundle IMG=<tag>`:
+```make bundle: ...
+  operator-sdk generate kustomize manifests -q
+  cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG) # Add this line
+  ...
+```
```

Comment:
````suggestion

```make
bundle: ...
  operator-sdk generate kustomize manifests -q
  cd config/manager && $(KUSTOMIZE) edit set image controller=$(IMG) # Add this line
  ...
```
````